J.Coate_H.Nguyen_Case_Study_1.Rmd

```diff
@@ -135,6 +145,8 @@ ggtitle("Median IBU by State") + ylab("IBU") + xlab("State")
 #### Themed graphs
 #### Labeles and titled graphs
 
+#Huy's comment:
+I don't want to create only 1 df for ABV and IBU because if we only consider the median of ABV, we only need to delete 62 NA and if we want to calculate IBU, we only need to delete 1005NA. That's why in my following codes for Question 4, I created 2 separated df, one for ABV and one for IBU.
```

Comment:
I just was wondering about what you wanted to discuss. The question is very weird and I want more clarity from our TA. It's on my list of things to ask tonight.